### PR TITLE
Update README.md with PR note

### PR DESCRIPTION
**Pull request by danielpeintner**
CR -> PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ information on how to create a pull request.
 
 __Note (August 2022):__
```
Comment (danielpeintner) [inline README.md:21]: I am unsure "why" the note still says August 2022 ?

Comment (egekorkan) [reply README.md:21]:
```suggestion
__Note (July 2023):__
```

File: README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ information on how to create a pull request.
...
 
-__Please notice that Thing Description 1.1 is in the review phase for the Candidate Recommendation (CR) transition. 
+__Please notice that Thing Description 1.1 is in the review phase for the Proposed Recommendation (PR) transition.
```
Comment (egekorkan) [inline README.md:23]:
```suggestion
__Please notice that Thing Description 1.1 is in the review phase for the Recommendation transition. 
```

We are done with PR transition so now we are in the rec transition phase.

File: README.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ information on how to create a pull request.
 
 __Note (August 2022):__
 
-__Please notice that Thing Description 1.1 is in the review phase for the Candidate Recommendation (CR) transition. 
+__Please notice that Thing Description 1.1 is in the review phase for the Proposed Recommendation (PR) transition. 
 In this and subsequent phase, only editorial changes (e.g., adding examples, additional explanations) and bug fixes can be applied to the [editorial draft](http://w3c.github.io/wot-thing-description/). Editorial changes should be labeled with 'Editorial' in the issue and corresponding PRs. New feature requests can not be considered and will be postponed to the next version of the Thing Description.__
```
Comment (egekorkan) [inline README.md:24]:
```suggestion
In this phase, only editorial changes (e.g., adding examples, additional explanations) and bug fixes can be applied to the [editorial draft](http://w3c.github.io/wot-thing-description/) if the whole Working Group agrees on it. Editorial changes should be labeled with 'Editorial' in the issue and corresponding PRs. New feature requests can not be considered and will be postponed to the next version of the Thing Description.__  
```

**Review by egekorkan — CHANGES_REQUESTED** [review]

**Comment by danielpeintner** [comment]
@egekorkan the proposed changes look fine 👍

**Pull request merged**